JobConfig/digitize/OnSpill_epilog.fcl

```diff
@@ -0,0 +1,24 @@
+# set the event timing for OnSpill
+physics.producers.EWMProducer.SpillType : 1
+# this next produces additional hit-level payload to support MC truth matching.
+# It produces functionally equivalent output as the standard module but is slower, so don't use this script for timing studies
+physics.producers.TTmakeSH.FilterHits : false
+physics.producers.TTmakePH.TestFlag : true
+physics.producers.TTmakePH.StrawHitSelectionBits : ["EnergySelection","TimeSelection","RadiusSelection"]
+physics.producers.TTmakePH.StrawHitMask          : ["Dead"]
+# add the trigger MC matching to the appropriate paths
+# apr
+physics.apr_highP : [@sequence::physics.apr_highP, TTAprKSFMC ]
```

Comment:
This is of course a point of by-hand maintenance as we discussed. `Mu2e/Production` will need to track when any new triggers of interest are added and if the `LoopHelixFit` instance changes in a given trigger path (e.g. if `apr_lowP_stopTarg_multitrk` changes to a new `LoopHelixFit` instance with lower momentum thresholds or different parameter constraints). I don't have any suggestions here though, just remarking on this point.